src/meta/processors/jobMan/JobManager.cpp

```diff
@@ -170,6 +181,27 @@ cpp2::ErrorCode JobManager::addJob(const JobDescription& jobDesc, AdminClient* c
     return cpp2::ErrorCode::SUCCEEDED;
 }
 
+size_t JobManager::jobSize() const {
+    return highPriorityQueue_->size() + lowPriorityQueue_->size();
+}
+
+bool JobManager::try_dequeue(JobID& jobId) {
+    if (highPriorityQueue_->try_dequeue(jobId)) {
+        return true;
+    } else if (lowPriorityQueue_->try_dequeue(jobId)) {
+        return true;
+    }
+    return false;
+}
+
+void JobManager::enqueue(const JobID& jobId, const cpp2::AdminCmd& cmd) {
+    if (cmd == cpp2::AdminCmd::STATIS) {
```

Comment:
seems statis job will take higher priority ?

Reply:
Yes, according to AdminCmd to determine whether it is high priority or low priority. 
Currently AdminCmd::STATIS is a high priority